UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Webjob.UnitTests/Services/FulfilmentAncillaryFilesTest.cs

```diff
@@ -372,10 +377,15 @@ public async Task WhenValidCreateLargeExchangeSetCatalogFileRequest_ThenReturnTr
         [Test]
```

Comment:
Are we actually testing the new code which has been added to the FulfilmentAncillaryFiles.cs? 

The bit we have added sits after....

if (listFulfilmentData != null && listFulfilmentData.Any())
           
I may have missed it but is this code block tested?

If not can we test it? (if this block fails it would alter the outcome of the method significantly)

Reply:
Resolved